arches_querysets/rest_framework/serializers.py

```diff
@@ -385,6 +385,9 @@ class ArchesTileSerializer(serializers.ModelSerializer, NodeFetcherMixin):
     resourceinstance = serializers.PrimaryKeyRelatedField(
         queryset=ResourceTileTree.objects.all(), required=False, html_cutoff=0
     )
+    nodegroup = serializers.PrimaryKeyRelatedField(
+        queryset=NodeGroup.objects.all(), required=False, html_cutoff=0
```

Comment:
html_cutoff=0 tells DRF not to do a bunch of queries to populate dropdowns in the browsable API html page.